### DISCUSSION TITLE
Github support team did not respond my ticket #3073448

### DISCUSSION THREAD

**Discussion by nhantruong97** [community/community#143473] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

We submitted a ticket five days ago, but no one has responded. Our GitHub Action isn’t working, causing a disruption in our workflow.

**Comment by thomassiard** [comment]
Here are some steps you can take to potentially speed up the process:

Check for Ticket Updates: Double-check your email for any updates or requests for more information from GitHub Support. Sometimes they may ask for details that can help them prioritize or resolve your issue faster.

Follow Up on the Ticket: Adding a follow-up message to your existing ticket can help keep it on their radar. Explain the impact on your workflow to underscore the urgency.

GitHub Status Page: Occasionally, GitHub Actions issues can be due to broader platform problems. Check [GitHub’s Status Page](https://www.githubstatus.com/) to see if there are any known incidents related to Actions.

Community Forum: In the meantime, you might find help in the [GitHub Community Forum](https://github.community/). Sometimes other users have faced similar issues, and you may find a workaround while waiting for official support.

**Comment by azwarnaim** [comment]
Can try check here : https://support.github.com/

**Comment by MatinGhanbari** [comment]
> <img alt="" width="32" height="32" src="https://avatars.githubusercontent.com/u/97091384?s=64&amp;v=4">
> nhantruong97

Don't worry these are the christmas days so that is why everything is blocked XD
I hope they will reply your ticket soon, best of luck :)
